include/crow/http_connection.h

```diff
@@ -186,15 +186,17 @@ namespace crow
           std::tuple<Middlewares...>* middlewares,
           std::function<std::string()>& get_cached_date_str_f,
           detail::task_timer& task_timer,
-          typename Adaptor::context* adaptor_ctx_):
+          typename Adaptor::context* adaptor_ctx_,
+          std::atomic<unsigned int>& queue_length_):
           adaptor_(io_service, adaptor_ctx_),
           handler_(handler),
           parser_(this),
           server_name_(server_name),
           middlewares_(middlewares),
           get_cached_date_str(get_cached_date_str_f),
           task_timer_(task_timer),
-          res_stream_threshold_(handler->stream_threshold())
+          res_stream_threshold_(handler->stream_threshold()),
+          queue_length(queue_length_)
```

Comment:
This should be inverted

Reply:
Why? It's the same order as the member declarations. Or you mean inverting `queue_length` and `queue_length_`

Reply:
No problem! Done